src/mappings/util.ts

```diff
@@ -247,3 +249,50 @@ export function addIfNotIncludes<T>(arr: T[], item: T): void {
     arr.push(item);
   }
 }
+
+type MeshPortfolio = {
```

Comment:
(nit) I think `interface` is preferred when possible. (LegDetails too)

If `Pick` doesn't work with interface then OK to leave, but I think it should.